submissions/jeff-gurpreet/scripts/models/repo.js

```diff
@@ -0,0 +1,33 @@
+(function(module) {
+  var repos = {};
+
+  repos.all = [];
+
+  repos.requestRepos = function(callback) {
+    /* TODO: How would you like to fetch your repos? Someone say AJAX!?
+       Don't forget to call the callback! */
+    $.ajax({
+      url: 'https://api.github.com/users/jeffgebhardt/repos' +
+          '?per_page=10' +
+          '&sort=updated',
+      type: 'GET',
+      headers: {'Authorization': 'token ' + myToken},
+      success: function(data, message, xhr) {
+        console.log(data);
+        repos.all = data;
```

Comment:
now that we have our data, we want to call the callback function to move on to the next step